penn/transit.py

```diff
@@ -65,20 +67,35 @@ def transapc(self, start_date, end_date):
             'end': self.formatDate(end_date)
         }
         response = self._request(ENDPOINTS['TRANSAPC'], params)
-        return response;
+        return response
 
-    def stopinventory(self, start_date, end_date):
+    def stopinventory(self):
         """Return a list of venue objects.
 
 
         >>> venues = din.venues()
         """
-        params = {
-            'start': self.formatDate(start_date),
-            'end': self.formatDate(end_date)
-        }
-        response = self._request(ENDPOINTS['STOPINVENTORY'], params)
-        return response;
+        response = self._request(ENDPOINTS['STOPINVENTORY'], None)
```

Comment:
2nd argument of `self._request` is `None` by default, so remove second argument (multiple times) to keep it clear.